packages/connect-core/src/entities/Role.ts

```diff
@@ -29,14 +29,16 @@ export default class Role extends CoreEntity implements RoleData {
     if (artifact) {
       const { roles }: AragonArtifact = parseMetadata(artifact, 'artifact.json')
 
-      const role = roles.find((role) => role.bytes === data.hash)
+      const role = roles.find(role => role.bytes === data.hash)
 
       this.name = role?.id
       this.description = role?.name
     }
 
     this.appAddress = data.appAddress
-    this.grantees = data.grantees
+    this.grantees = data.grantees?.map(
```

Comment:
I decided to create a `Permission` instance for every grantee entity.

Reply:
Should we name it `permissions` then?